src/core/referencingfeaturelistmodel.h

```diff
@@ -162,8 +163,8 @@ class QFIELD_CORE_EXPORT ReferencingFeatureListModel : public QAbstractItemModel
     void modelUpdated();
 
   private slots:
-    void updateModel();
-    void gathererThreadFinished();
+    virtual void updateModel();
+    virtual void gathererThreadFinished();
```

Comment:
I meant to ask the same thing as for the delete for these two methods, would it be possible to delegate the specific behavior to a different method instead of overriding them as a whole?

Reply:
Should be good now. I guess the requested change is mostly due to the good practice to acoid public virtual methods/slots. Out of curiosity, except the potential risk of error/code duplication, is there deeper technical reason for that?

Reply:
If you fix something in the parent method, or want to add some special behavior that always happens before or after, it's impossible to do this way.